src/platform/packages/private/kbn-ui-shared-deps-src/kibana.jsonc

```diff
@@ -1,9 +1,9 @@
 {
-  "type": "shared-common",
+  "type": "shared-browser",
```

Comment:
These changes trickled around from updating the type declaration of the error_boundary package